src/Aspire.Hosting/Seq/SeqBuilderExtensions.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Seq server resources to the application model.
+/// </summary>
+public static class SeqBuilderExtensions
+{
+    // The path within the container in which Seq stores its data
+    const string SeqContainerDataDirectory = "/data";
+
+    /// <summary>
+    /// Adds a Seq server resource to the application model. A container is used for local development.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name to give the resource.</param>
+    /// <param name="port">The host port for the Seq server.</param>
+    /// <param name="seqDataDirectory">Host directory to bind to Seq's data directory. This must already exist.</param>
+    public static IResourceBuilder<SeqResource> AddSeq(
+        this IDistributedApplicationBuilder builder,
+        string name,
+        int? port = null,
+        string? seqDataDirectory = null)
+    {
+        var seqResource = new SeqResource(name);
+        var resourceBuilder = builder.AddResource(seqResource)
+            .WithHttpEndpoint(hostPort: port, containerPort: 80)
+            .WithAnnotation(new ContainerImageAnnotation {Image = "datalust/seq", Tag = "latest"})
```

Comment:
Use the new `WithContainerImage` method? Do you want to use latest by default? We've moved to specifying versions in these methods for stablilty.

Reply:
I can't find `WithContainerImage`. I found `AddContainer` which would be good except that it uses `ContainerResource` and then I don't know how to find its endpoint address from the component side.

Reply:
Oops 
https://github.com/dotnet/aspire/blob/733fbd2ce488b1058f0723b56258802c606537dd/src/Aspire.Hosting/Extensions/ContainerResourceBuilderExtensions.cs#L106-L140

Reply:
Thanks. The doc comment for the `image` param on that last overload of `WithImage` looks wrong to me.

Reply:
Thanks! Yea, you're right.